packages/p2p/src/components/buy-sell/__tests__/buy-sell.spec.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { useStores } from 'Stores/index';
+import BuySell from '../buy-sell';
+
+const mock_store = {
+
+    general_store: {
+        should_show_popup: false,
+    },
+    buy_sell_store: {
+        registerIsListedReaction: jest.fn(),
+        registerAdvertIntervalReaction: jest.fn(),
+        selected_local_currency: 'USD',
+        show_advertiser_page: false,
+        should_show_verification: true,
+        setLocalCurrency: jest.fn(),
+    },
+};
```

Comment:
@nada-deriv P2P stores are accessible inside the P2P package and we don't have the root store problem, So instead of mocking them, we can simply create instances of the stores in the test files.